include/picongpu/algorithms/FieldToParticleInterpolation.hpp

```diff
@@ -29,6 +29,26 @@
 namespace picongpu
 {
 
+template<typename T_Type>
+struct GetComponent
+{
+    using Type = T_Type;
+    using result_type = Type;
+    uint32_t component;
+
+    HDINLINE GetComponent(const GetComponent&) = default;
```

Comment:
Maybe this constructor can be removed. see [this](https://github.com/ComputationalRadiationPhysics/picongpu/pull/2150/files#diff-54b2a2217632dae0dd703dfbdf37b113R59) implementation